### PR TITLE
Add type and group to winum-auto-setup-mode-line

### PR DESCRIPTION
**Pull request by rgrinberg**
fixes warnings

### CODE REVIEW

File: winum.el [hunk 1]
```diff
@@ -97,7 +97,9 @@ Example: always assign *Calculator* the number 9 and *NeoTree* the number 0:
 (defcustom winum-auto-setup-mode-line t
   "When nil, `winum-mode' will not display window numbers in the mode-line.
 You might want this to be nil if you use a package that already manages window
-numbers in the mode-line.")
+numbers in the mode-line."
+  :group 'winum
+  :type 'boolean)
```
Comment (deb0ch) [inline winum.el]: Could you please add a space after the `:type`, to have them aligned like I did for the other `customize` options ? :smile_cat:

**Comment by rgrinberg** [comment]
Updated

**Comment by deb0ch** [comment]
Thanks ! Nice catch btw ! :thumbsup:

Although I have never seen such warnings (curious of how you got them ?) and `customize` is so smart that this setting still appears in the `winum` customization group :open_mouth:, at least we now have `customize`'s nice `Toggle` mechanism for this one :smiley_cat: 

Anyways, I'm merging it. Thanks for this PR !  :tada::champagne:

**Comment by rgrinberg** [comment]
The warning are compilation warnings. If you byte compile the file you should see them. I run emacs from master though, so you might need something very recent.

**Comment by deb0ch** [comment]
That must be it then, thanks for the info. I'm still on 24.5 :sweat_smile: 

Will get to Emacs 25 with my upcoming OS change, soonish.

**Pull request merged**